clock_exercise/clockmanager.h

```diff
@@ -34,4 +55,8 @@ private slots:
 private:
     void refreshTime();
     void refreshDateText();
+    void refreshTimerHoursAndMinutes();
+    void decreaseTimerCurrentSeconds();
+    float convertHoursAndMinutesToSeconds(int hours, int minutes);
+    void noneExp(){};
```

Comment:
Non mi fa proprio impazzire il fatto che usiamo questa funzione vuota nel caso in cui la condizione del ternario sia falsa. Perchè non usiamo semplicemente un `if (timerCurrentSecondsNotifier.isChanged(m_timerCurrentSeconds)) emit timerCurrentSecondsChanged();`?

Reply:
È stato già corretto in un branch successivo, per il momento lo lascio così